src/site/tools/pub/pubspec.markdown

```diff
@@ -70,7 +70,7 @@ dependencies:
   efts: '>=2.0.4 <3.0.0'
   transmogrify: '>=0.4.0 <1.0.0'
 dev_dependencies:
-  unittest: '>=0.6.0 <0.12.0'
+  test: '>=0.6.0 <0.12.0'
```

Comment:
It seems like a losing battle to try to keep this consistently up-to-date. I'd probably just leave it as-is.